prep-html-syntax/index.html

```diff
@@ -0,0 +1,10 @@
+<!DOCTYPE html>
+<html>
+  <head>
+<h1 style="color:darkolivegreen;">My favorite Star Wars character is Yoda</h1>
+  </head>
```

Comment:
Hi Nadia, be sure to put the `<h1>` in the body of your document. We reserve the `<head>` for the `<title>` and other metadata.